sharedresource/v1alpha1/0000_10_sharedresource.crd.yaml

```diff
@@ -0,0 +1,112 @@
+# this is the boilerplate crd def that controller-gen reads and modifies with the
+# contents from share_type.go
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: shares.sharedresource.openshift.io
```

Comment:
@adambkaplan Should this be `csi.sharedresource.openshift.io` instead, or is this correct?

Reply:
This is fine - IMO preferable. The CSI aspect is really an implementation detail.